modules/page_object_about_prefs.py

```diff
@@ -322,3 +322,40 @@ def get_iframe(self) -> WebElement:
         Gets the webelement for the iframe that commonly appears in about:preferences
         """
         return self.get_element("browser-popup")
+
+    def set_etp_custom_mode(self) -> BasePage:
+        """Select the Custom mode in Enhanced Tracking Protection section"""
+        self.get_element("custom-radio").click()
+        return self
+
+    def set_etp_custom_mode_options(self, options_to_check: List[str]) -> BasePage:
+        """
+        Configures the Custom mode settings in the Enhanced Tracking Protection (ETP) section by selecting
+        specified options.
+
+        This method first ensures that the browser is set to Custom mode. It then deselects all available
+        tracking protection options (there are checked by default) and selectively re-checks only those specified in the
+        `options_to_check` list (multiple values can be checked at once).
+        """
+        self.set_etp_custom_mode()
+        checkbox_mapping = {
```

Comment:
I believe the prefs you need are these, though I think I have the wrong values for what you are trying to do, I'm just letting you know if you want to complete with a fixture for prefs u have the choice! :
```
@pytest.fixture()
def add_prefs():
    return [
        ("network.cookie.cookieBehavior", 0),
        ("privacy.trackingprotection.pbmode.enabled", False),
        ("privacy.trackingprotection.cryptomining.enabled", False),
        ("privacy.trackingprotection.fingerprinting.enabled", False),
        ("privacy.fingerprintingProtection.pbmode", False),
    ]
```